modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -194,8 +194,12 @@ The `hosts` parameter is a list of separate bare metal assets used to build the
 
 | `bootMACAddress`
 |
-| The MAC address of the NIC that the host uses for the `provisioning` network. Ironic retrieves the IP address using the `bootMACAddress` configuration setting. Then, it binds to the host.
+a| The MAC address of the NIC that the host uses for the `provisioning` network. Ironic retrieves the IP address using the `bootMACAddress` configuration setting. Then, it binds to the host.
```

Comment:
Hi @EricPonvelle I need to use the [ a operator](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator) to allow the note to render  correctly. Without the operator, it would not render as expected.